helm/cluster-eks/templates/_managed_machine_pools.tpl

```diff
@@ -37,13 +38,18 @@ metadata:
   name: {{ include "resource.default.name" $ }}-{{ $name }}
   namespace: {{ $.Release.Namespace }}
 spec:
+  additionalTags:
+    {{- if $.Values.global.providerSpecific.additionalResourceTags -}}{{- toYaml $.Values.global.providerSpecific.additionalResourceTags | nindent 4 }}{{- end}}
+    k8s.io/cluster-autoscaler/enabled: "true"
+    k8s.io/cluster-autoscaler/{{ include "resource.default.name" $ }}: "true"
+    giantswarm.io/cluster: {{ include "resource.default.name" $ }}
```

Comment:
Do we need the extra tags as well? AFAIK the tags specified https://github.com/giantswarm/cluster-eks/blob/main/helm/cluster-eks/templates/_managed_control_plane.tpl#L17-L18 should propagate to all AWS resources and these settings only add additional tags on top of that. SO I think this will either fail as duplicate tags or not have any effect.
 
 I am talking about:
 ```
 {{- if $.Values.global.providerSpecific.additionalResourceTags -}}{{- toYaml $.Values.global.providerSpecific.additionalResourceTags | nindent 4 }}{{- end}}
 giantswarm.io/cluster: {{ include "resource.default.name" $ }} 
 ```

Reply:
I might be wrong tho as this is just an assumption and the implementation might have bugs.


If you can manage to test it  and see if there is a difference that would be awesome.

Reply:
Sure, doing some tests now!

Reply:
You are right, additional tags (.Values.global.providerSpecific.additionalResourceTags) are propagated from the controlplane to the nodepools!
Setting them also on the machinepool causes no issues (no duplicates and no fails), but since it's redundant I will remove those and just leave the autoscaler tags.

Reply:
Thanks for testing!